massa-models/src/operation.rs

```diff
@@ -972,13 +970,13 @@ impl SecureShareOperation {
     }
 
     /// Get the max amount of gas used by the operation (`max_gas`)
-    pub fn get_gas_usage(&self) -> u64 {
+    pub fn get_gas_usage(&self, gas_usage: u64) -> u64 {
```

Comment:
* rename "gas_usage" to "base_operation_gas_cost" to be uniform with the settings
* also apply `base_op_gas_usage` to executesc and callsc: `max_gas.saturating_add(base_op_gas_usage)`